Jenkinsfile

```diff
@@ -127,8 +127,7 @@ def is_authorized(name) {
               'kkranen',
               'bgawrych', 'itaraban', 'daniil-sizov', 'anko-intel', 'Kacper-Pietkun',
               'hankaj', 'agrabows', 'DominikaJedynak', 'RafLit',
-              'VoVAllen',
-              'keli-wen',
+              'VoVAllen', 'keli-wen', 'jingyu-cao'
```

Comment:
pls verify it takes effect as expected.

Reply:
I think this should be your GitHub id here.  @caojy1998